pkg/tfgen/docs.go

```diff
@@ -247,7 +247,7 @@ func mergeDocs(g *generator, info tfbridge.ResourceOrDataSourceInfo, org string,
 // nolint:lll
 var (
 	argumentBulletRegexp = regexp.MustCompile(
-		"\\*\\s+`([a-zA-z0-9_]*)`\\s+(\\([a-zA-Z]*\\)\\s*)?[–-]?\\s+(\\([^\\)]*\\)\\s*)?(.*)")
+		"\\*\\s+`([a-zA-z0-9_]*)`\\s*(\\([a-zA-Z]*\\)\\s*)?[–-]?\\s+(\\([^\\)]*\\)\\s*)?(.*)")
```

Comment:
Possibly worth adding tests to cover the desired behaviour of this regexp?  Potentially including a bunch of examples from real docs to ensure they are handled correctly?  (I realize this is existing test debt, but might be a good opportunity to improve on that while we are tweaking this).